cpp/tests/utilities/cudf_test_utils.cuh

```diff
@@ -223,7 +286,8 @@ std::vector<gdf_col_pointer> initialize_gdf_columns(
  * @return bool Whether or not the columns are equal
  * ---------------------------------------------------------------------------**/
 template <typename T>
-bool gdf_equal_columns(gdf_column* left, gdf_column* right) {
+bool gdf_equal_columns(gdf_column* left, gdf_column* right)
+{
```

Comment:
Please don't make changes that *only* move brackets around. Bracket placement is not dictated by a coding standard so you are only wasting reviewers' time.

Reply:
Ok, but - this _was_ a no-functionality-changes commit, and indentation fixes seem like they go well with include reordering - very minor stuff..